### PR TITLE
AGFEO Dashboard 2.0.12 #168347

### PR DESCRIPTION
**Pull request by agfeo-rw**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/168349)

### CODE REVIEW

**Comment by agfeo** [comment]
Sorry, my system is set to German language. The output message of `widget validate` says, that the check ended successfully:
```bash
PS Z:\Development\work\winget-pkgs\manifests\a\AGFEO\AGFEODashboard\2.0.12> winget validate --manifest .
Die Manifestüberprüfung war erfolgreich.
PS Z:\Development\work\winget-pkgs\manifests\a\AGFEO\AGFEODashboard\2.0.12>
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334957
> 2024-08-20 09:17:27.861 [CLI ] ShellExecute installer failed: 1633
> MSI (s) (50:48) [09:17:27:624]: Product: AGFEO Dashboard -- Installation failed.
> This installation package is not supported by this processor type. Contact your product vendor.

Automated error analysis suggests -1978334957 may mean Package not supported by the system.

Automated error analysis suggests 1633 may mean `ERROR_INSTALL_PLATFORM_UNSUPPORTED` - This installation package is not supported by this processor type. Does this manifest have `Architecture: x86`? It might need to be `x64`.

(Automated response - build 887.)

File: manifests/a/AGFEO/AGFEODashboard/2.0.12/AGFEO.AGFEODashboard.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AGFEO.AGFEODashboard
+PackageVersion: 2.0.12
+InstallerLocale: de-DE
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/a/AGFEO/AGFEODashboard/2.0.12/AGFEO.AGFEODashboard.installer.yaml:11]:
```suggestion
- Architecture: x64
```

**Review by agfeo-rw — COMMENTED** [review]
I accepted the suggested change in `manifests/a/AGFEO/AGFEODashboard/2.0.12/AGFEO.AGFEODashboard.installer.yaml`
Property is now set to `Architecture: x64`

**Review by stephengillie — APPROVED** [review]

**Pull request merged**